sig/handwritten/app/models/announcement.rbs

```diff
@@ -9,5 +9,12 @@ class Announcement < ApplicationRecord
     #       It should be removed if rbs_rails will support it.
     #       refs: https://github.com/pocke/rbs_rails/pull/268
     def published: () -> self
+
+    # NOTE: Added by ActionText
+    def with_rich_text_content_and_embeds: () -> self
+
+    # NOTE: Added by kaminari
+    def page: (untyped) -> self
+    def per: (Integer) -> self
```

Comment:
Kaminari internally adds methods `#per` and `#page` to the ActiveRecord
relation objects.  At this moment, it is difficult to give types for
such 3rd party gems extending ActiveRecord models.

Therefore this adds types for these methods to the each subclasses using
pagination via kaminari.